snuba/settings/__init__.py

```diff
@@ -49,6 +55,18 @@
         },
         "single_node": True,
     },
+    {
+        "host": os.environ.get("CLICKHOUSE_PROFILING_HOST", CLICKHOUSE_HOST),
+        "port": int(os.environ.get("CLICKHOUSE_PROFILING_PORT", CLICKHOUSE_PORT)),
+        "user": os.environ.get("CLICKHOUSE_PROFILING_USER", CLICKHOUSE_USER),
+        "password": os.environ.get("CLICKHOUSE_PROFILING_PASSWORD", CLICKHOUSE_PASSWORD),
+        "database": os.environ.get("CLICKHOUSE_PROFILING_DATABASE", CLICKHOUSE_DATABASE),
+        "http_port": int(os.environ.get("CLICKHOUSE_PROFILING_HTTP_PORT", CLICKHOUSE_HTTP_PORT)),
+        "storage_sets": {
+            "stacktraces",
+        },
+        "single_node": True,
+    },
```

Comment:
Do you mind just keeping these all on the same cluster instead of adding a new one? This default configuration is used in self hosted and is intentionally as simple as possible with all the storage sets assigned to one cluster.

Reply:
For sure, I'll remove that.